app/src/main/java/com/sapuseven/untis/activities/BaseActivity.kt

```diff
@@ -18,7 +18,7 @@ import java.io.File
 
 @SuppressLint("Registered") // This activity is not intended to be used directly
 open class BaseActivity : AppCompatActivity() {
-	private var currentTheme: String = ""
+	var currentTheme: String = ""
```

Comment:
I'd prefer this to be `protected` as you only need the value in subclasses.
```suggestion
	protected var currentTheme: String = ""
```

Reply:
Yes that makes sense